INTEGRATING.md

```diff
@@ -1,34 +1,103 @@
-## Integrating Loki
+## Integrating Loki Charmed Operator
 
-Loki integrates with the following:
+### Overview
 
+This Loki Charmed Operator provides two libraries so you can use them to integrate it
+with other charms.
 
-1. Any charm that supports the `loki_push_api` interface.
-You can integrate it using the Loki Charm library. The documentation of
-the charm library, available through [Charmhub](https://charmhub.io/loki-k8s)
-provides further details. The information
-exchanged through the `loki_push` interface can be broken down
-into two parts
+- [`loki_push_api`](https://charmhub.io/loki-k8s/libraries/loki_push_api)
+- [`log_proxy`](https://charmhub.io/loki-k8s/libraries/log_proxy)
 
 
-- Loki charm provides an [endpoint URL](https://grafana.com/docs/loki/latest/api/#post-lokiapiv1push)
-to receive log from [Loki clients](https://grafana.com/docs/loki/latest/clients/) that
-relates with this charm.
+The library `loki_push_api` provides
+[endpoint URL](https://grafana.com/docs/loki/latest/api/#post-lokiapiv1push)
+to receive log from [Loki clients](https://grafana.com/docs/loki/latest/clients/) that relates
+with this charm.
 
-- Loki may receive alert rules which tell when to raise alerts. These
-  rules are read from a directory named `loki_alert_rules`, if
-  present at the top level, within the client charm's source
-  (`src`) directory.
+Besides this Loki charmed operator may receive from other charms alert rules which tells when
+to raise alerts.
+These rules are read from a directory named `loki_alert_rules`, if present at the top level,
+within the client charm's source (`src`) directory.
 
 Any time new relations are made, or existing ones changed, between the
-Loki charm and any scrape target charm, a new Loki
-configuration is generated.
+Loki charm and any scrape target charm, a new Loki configuration is generated.
 
-2. Loki integrates with
+The library `log_proxy` provides a way to setup a charms as a Log Proxy (Provider or Consumer) to Loki.
+
+### Libraries
```

Comment:
Should the content below be moved into the "code overview" section?

Reply:
What do you think @simskij ?